st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -62,11 +61,6 @@ def post(self, action_alias):
             LOG.exception('Validation failed for action alias data=%s.', action_alias)
             pecan.abort(http_client.BAD_REQUEST, str(e))
             return
-        except StackStormDBObjectConflictError as e:
-            LOG.warn('ActionAlias creation of %s failed with uniqueness conflict.', action_alias,
```

Comment:
Yes, this means we don't have "custom per controller method" log message anymore, but we still log the error inside the JSONErrorResponseHook and those messages are now unified.

On top of that, we now include additional context there (https://github.com/StackStorm/st2/pull/1738/files#diff-545d9d9f5fecdd988de6c2d684fe7a89R170) which should make it easier to find relevant errors, especially if you are used structured logging and something like Graylog2 / loggly (which everyone should be using in production deployments anyway).